pkg/systemd/services/timers.scss

```diff
@@ -12,4 +12,8 @@
     .specific-repeat-group, .delay-group {
         row-gap: var(--pf-global--spacer--sm);
     }
+
+    .custom-group {
+        width: 50rem;
+    }
```

Comment:
why? This seems fragile on different screen sizes and also I think it broke it for me right now

Reply:
Yeah, we shouldn't have hardcoded sizes like this. What is "custom-group"? The naming doesn't even say what it's for.

Reply:
@garrett and @marusak Thanks for the suggestions.  First time working on cockpit so still trying to figure out your standard practices.  I added the css as I didn't like the icon location trailing underneath the input.  but since we'll switch to helper text that issue goes away.  however it still exists for the delay-group and will need to addressed at a later date to stay away from hard coded sizes.